validation/fleet/snapshot/snapshot_restore_existing_gitrepo_test.go

```diff
@@ -35,11 +35,10 @@ const (
 
 type FleetWithSnapshotTestSuite struct {
 	suite.Suite
-	client         *rancher.Client
-	session        *session.Session
-	fleetGitRepo   *v1alpha1.GitRepo
-	clustersConfig *etcdsnapshot.Config
```

Comment:
clustersConfig was removed from fleet file. is that intentional?

Reply:
yeah, it isn't used. Thanks golangci-lint!